htdocs/src/Oc/Page/PageEntity.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Oc\Page;
+
+use DateTime;
+
+/**
+ * Class PageEntity
+ *
+ * @package Oc\Page
+ * @author Nick Lubisch <oc@nicklubisch.de>
+ */
+class PageEntity
+{
+    /**
+     * @var int
+     */
+    public $id;
+
+    /**
+     * @var string
+     */
+    public $slug;
+
+    /**
+     * @var string
+     */
+    public $metaKeywords;
+
+    /**
+     * @var string
+     */
+    public $metaDescription;
+
+    /**
+     * @var string
+     */
+    public $metaSocial;
+
+    /**
+     * @var DateTime
+     */
+    public $updatedAt;
+
+    /**
+     * @var bool
+     */
+    public $active = true;
+
+    /**
+     * Checks if the entity is new.
+     *
+     * @return bool
+     */
+    public function isNew()
+    {
+        return $this->id === null;
+    }
+
+    /**
+     * Sets properties from the database.
+     *
+     * @return array
+     */
+    public function toDatabaseArray()
+    {
+        return [
+            'id' => $this->id,
+            'slug' => $this->slug,
+            'meta_keywords' => $this->metaKeywords,
+            'meta_description' => $this->metaDescription,
+            'meta_social' => $this->metaSocial,
+            'updated_at' => $this->updatedAt->format(DateTime::ATOM),
+            'active' => $this->active
+        ];
+    }
+
+    /**
+     * Prepares database array from properties.
+     *
+     * @param array $data
+     *
+     * @return self
+     */
+    public function fromDatabaseArray(array $data)
+    {
+        $this->id = (int) $data['id'];
+        $this->slug = (string) $data['slug'];
+        $this->metaKeywords = (string) $data['meta_keywords'];
```

Comment:
I think we get only string values from database so the type cast to string is not needed or?